src/modules/addToUI.js

```diff
@@ -0,0 +1,63 @@
+import ToDo from './todo.js';
+// Add items to UI
+const addToList = () => {
+  const todoList = document.getElementById('todo-list');
+  todoList.innerHTML = '';
+
+  ToDo.list.forEach((item) => {
+    const listItem = document.createElement('li');
+    listItem.setAttribute('id', item.index);
+    listItem.classList = 'item-container';
+
+    listItem.innerHTML = `
+    <input class="checkbox" type="checkbox">
+    <input type="text" class="input" value='${item.description}'>
+    <button class="cancel-btn"><i class="fa fa-trash" aria-hidden="true"></i></button>
+    `;
+
+    // <p class="text-area">${item.description}</p>
+
+    todoList.appendChild(listItem);
+
+    const checkbox = listItem.querySelector('input');
+    const text = listItem.querySelector('.input');
+    const textInput = listItem.querySelector('p');
+    const deleteButton = listItem.querySelector('button');
+
+    // Update
+    checkbox.addEventListener('change', () => {
+      const index = parseInt(listItem.id, 10);
+      ToDo.list[index].update();
+      text.classList.toggle('complete');
+      textInput.classList.toggle('complete');
+      localStorage.setItem('todoList', JSON.stringify(ToDo.list));
+    });
+
+    text.addEventListener('input', (e) => {
+      text.value = e.target.value;
+      const index = parseInt(listItem.id, 10);
+      ToDo.list[index].description = e.target.value;
+      localStorage.setItem('todoList', JSON.stringify(ToDo.list));
+      if (e.code === 'Enter') {
+        // text.style.display = 'block';
```

Comment:
Do not commit old pieces of code as inline comments. They will make your project look messy.